lms/djangoapps/instructor/views/legacy.py

```diff
@@ -730,14 +743,30 @@ def get_student_grade_summary_data(
                     # value as a float between 0 and 1, or a two-tuple
                     # containing the earned score and possible score for
                     # the assignment (see docstring).
+                    gradeset = student_grades(
+                         student, request, course, keep_raw_scores=get_raw_scores, use_offline=use_offline
+                    )
+                    log.debug(u'student=%s, gradeset=%s', student, gradeset)
```

Comment:
Is there a reason why we use student_grades in this code branch progress_summary in the other code branch?

Can't we use progress_summary for both?